### ISSUE TITLE
In CI check Swagger based server and client 

### ISSUE THREAD

**Issue by metalmatze** [gitpods/gitpods#28]
Before running anything else Go and Dart related, run the Swagger generation for both and then check for any differences by running `git diff --exit-code`.
This will fail if the currently committed server or client are outdated compared to swagger.yaml

**Comment by tboerger** [comment]
We could add a preflight pipeline that generates all and fails early if there are uncommitted changes.

**Comment by metalmatze** [comment]
I think prepending these two steps (because we need to use two different images for the generators) to the testing pipeline is enough for now.

Take a look at the `Makefile`, there you can see which images are used. I already added `$(SWAGGER)` and `$(GOSWAGGER)` to replace those in the CI with the direct binaries. 

Do you want to work on that, @tboerger? :rela

**Comment by tboerger** [comment]
I know that we need two different images, sadly we don't generate it from one tool :)

**Comment by metalmatze** [comment]
Yes, but to be quite honest, while working on this last week it never was an issue. Of course it once has to pull 2 images, but after that it's just `make generate` 😊😉
